Dockerfile

```diff
@@ -1,26 +1,36 @@
 FROM heroku/cedar:14
 MAINTAINER Terence Lee <terence@heroku.com>
 
+ENV RUBY_MAJOR 2
+ENV RUBY_MINOR 3
+ENV RUBY_PATCH 0
```

Comment:
I don't think it's a good idea to have ENV vars for major minor and patch versions, since this adds extra stos to build.

Just the `RUBY_VERSION` ENV is enough. Neither `RUBY_BASE` is really necessary.